src/game_io/game_player_info_packet.cc

```diff
@@ -82,6 +82,10 @@ void GamePlayerInfoPacket::read(FileSystem& fs, Game& game, MapObjectLoader*) {
 					player->msites_defeated_ = fr.unsigned_32();
 					player->civil_blds_lost_ = fr.unsigned_32();
 					player->civil_blds_defeated_ = fr.unsigned_32();
+
+					for (size_t j = (packet_version >= 24 ? fr.unsigned_32() : 0); j; --j) {
```

Comment:
I think this is a bit obscure, and it's not immediately clear from the code how often the size limit will be read from the packet.

I suggest something like this:

```
if (packet_version >= 24) {
    const size_t no_of_muted = fr.unsigned_32();
    for (size_t j =  0); j < no_of_muted; ++j) {
        ...
    }
}
```

Reply:
done